src/scripts/main.js

```diff
@@ -1 +1,62 @@
 'use strict';
+
+document.addEventListener('contextmenu', (e) => e.preventDefault());
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('mousedown', (e) => {
+    if (e.button === 0) {
+      resolve('First promise was resolved');
+    }
+  });
+
+  setTimeout(() => reject(new Error('First promise was rejected')), 3000);
+});
+
+const secondPromise = new Promise((resolve) => {
+  document.addEventListener('mousedown', (e) => {
+    if (e.button === 0 || e.button === 2) {
+      resolve('Second promise was resolved');
+    }
+  });
+});
+
+let leftClicked = false;
+let rightClicked = false;
+
+const thirdPromise = new Promise((resolve) => {
+  document.addEventListener('mousedown', (e) => {
+    if (e.button === 0) {
+      leftClicked = true;
+    }
+
+    if (e.button === 2) {
+      rightClicked = true;
+    }
+
+    if (leftClicked && rightClicked) {
+      resolve('Third promise was resolved');
+    }
+  });
+});
+
+function successHandler(message) {
+  const div = document.createElement('div');
+
+  div.setAttribute('data-qa', 'notification');
+  div.className = 'success';
+  div.textContent = message;
+  document.body.append(div);
+}
+
+function errorHandler(error) {
+  const div = document.createElement('div');
+
+  div.setAttribute('data-qa', 'notification');
+  div.className = 'error';
+  div.textContent = error.message;
+  document.body.append(div);
+}
+
+firstPromise.then(successHandler).catch(errorHandler);
+secondPromise.then(successHandler).catch(errorHandler);
```

Comment:
The second promise does not have a rejection handler. Although it currently cannot reject, it's a good practice to include a catch block for consistency and future-proofing.